server/src/main/java/au/org/aodn/ogcapi/server/core/model/RecordSuggestDTO.java

```diff
@@ -1,16 +1,25 @@
 package au.org.aodn.ogcapi.server.core.model;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import java.util.List;
+import java.util.Map;
+
 @Data
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class RecordSuggestDTO {
-    private String title;
+    private List<String> abstractPhrases;
+
+    @JsonProperty("record_suggest")
```

Comment:
Not easy to understand this, is that you want to execute something before json transform, why not do it in constructor?

Reply:
It was this problem and instruction: https://stackoverflow.com/a/41383076 @utas-raymondng

otherwise will just get null in the mapped hits()

Reply:
I added a comment if that will be enough.